components/movementsensor/imuwit/imu.go

```diff
@@ -1,22 +1,38 @@
 // Package imuwit implements wit imus.
-// Tested on the HWT901B and BWT901CL models. Other WT901-based models may work too.
 package imuwit
 
+/*
+Sensor Manufacturer:  		Wit-motion
+Supported Sensor Models: 	HWT901B, BWT901, BWT61CL
+Supported OS: Linux Debian GNU/Linux 11 (bullseye)
```

Comment:
I cannot get the serial device path to show up on a mac. Made a ticket to follow up, it's showing up on Linux (debian and ubuntu) and Windows.

Reply:
If it works on all Linuxes you've tried it on, I'd at least remove the mention of Bullseye. Otherwise, we imply that it only works on the rpi and beagle.